airbyte-oauth/src/test/java/io/airbyte/oauth/flows/BaseOAuthFlowTest.java

```diff
@@ -197,6 +197,18 @@ protected String getConstantState() {
     return "state";
   }
 
+  protected String getMockedResponse() {
+    final Map<String, String> returnedCredentials = getExpectedOutput();
+    return Jsons.serialize(returnedCredentials);
+  }
+
+  protected OAuthConfigSpecification getOAuthConfigSpecification() {
+    return getoAuthConfigSpecification()
+        // change property types to induce json validation errors.
+        .withCompleteOauthServerOutputSpecification(getJsonSchema(Map.of("client_id", Map.of("type", "integer"))))
```

Comment:
@etsybaev  I was sure that client_id and secret should be a 'string'? Could you please re-check